README.md

```diff
@@ -105,6 +105,7 @@ The package includes the following rules.
 | [no-subclass](docs/rules/no-subclass.md)                               | Disallow subclassing RxJS classes.                                                                        | 🔒   |    |    | 💭 |    |
 | [no-subject-unsubscribe](docs/rules/no-subject-unsubscribe.md)         | Disallow calling the `unsubscribe` method of subjects.                                                    | ✅ 🔒 |    |    | 💭 |    |
 | [no-subject-value](docs/rules/no-subject-value.md)                     | Disallow accessing the `value` property of a `BehaviorSubject` instance.                                  |      |    |    | 💭 |    |
+| [no-subscribe-in-pipe](docs/rules/no-subscribe-in-pipe.md)             | Forbids the calling of `subscribe` within any operator inside a `pipe` operation.                         | ✅ 🔒 |    |    |   |    | 
```

Comment:
To add this rule to the `recommended` and `strict` configurations, you'll need to:

1. Update `configs/recommended.ts` and `configs/strict.ts`
2. Set the `"meta.docs.recommended"` property to indicate it's part of the recommended config (and strict too, which always includes all recommended rules): `recommended: 'recommended'`
3. Run `yarn docs` to auto-update the markdown documentation with badges indicating which configs the rule is part of.

Reply:
Added the rule to the strict and recommended config files and then ran `yarn docs`